hasher-matcher-actioner/scripts/submit_content_test.py

```diff
@@ -134,16 +140,47 @@ def set_up_basic_test(self, hostname: str, port: int):
             action_rule=action_rule,
         )
 
-    def _submit_for_basic_test(self):
-        content_id = "submit_content_test_id_1"
-        filepath = "sample_data/b.jpg"
-        additional_fields = ["i-am:a-bridge", "this-is:a-test"]
-        with open(filepath, "rb") as file:
-            self.api.send_single_submission_b64(
-                content_id,
-                str(base64.b64encode(file.read()), "utf-8"),
-                additional_fields,
-            )
+    def clean_up_test(self, hostname: str, port: int):
+        """
+        Deletes specific action and action rules
+        """
+        raise NotImplementedError
+
+    def submit_test_content(
+        self,
+        content_id="submit_content_test_id_1",
+        filepath="sample_data/b.jpg",
+        additional_fields=[
+            "this-is:a-test",
+            "submitted-from:submit_content_test.py",
+        ],
+    ):
+        try:
+            with open(filepath, "rb") as file:
+                self.api.send_single_submission_url(
+                    content_id,
+                    file,
+                    additional_fields,
+                )
+        except urllib3.exceptions.MaxRetryError as errmr:
```

Comment:
nit: There is no real change in how things are handled based on the error, use parenthesis to capture all at once?

https://docs.python.org/3/tutorial/errors.html#handling-exceptions